waterloop-site/src/components/SustainableTech/AboutEvent/DetailsCard.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import styled from 'styled-components';
+import CalendarIcon from './img/calendar.svg';
+import ClockIcon from './img/clock.svg';
+import LocationIcon from './img/location.svg';
+
+const CardContainer = styled.div`
+  margin: 20px;
+  width: 345px;
+  background-color: white;
+  border-radius: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding-bottom: 20px;
+  box-shadow: 0px 3px 20px #d7e7f3;
+
+  @media screen and (max-width: 425px) {
+    width: 225px;
+  }
+`;
+
+const InfoContainer = styled.div`
+  min-width: 200px;
+`;
+
+const Title = styled.h1`
+  font-size: 25px;
+  margin-top: 20px;
+  color: #203d7a;
+
+  @media screen and (max-width: 768px) {
+    font-size: 20px;
+  }
+`;
+
+const Description = styled.p`
+  font-size: 18px;
+  margin: 5px 10px 10px 10px;
+  color: #737b7d;
+  text-align: center;
+
+  @media screen and (max-width: 768px) {
+    font-size: 16px;
+  }
+`;
+
+const IconCircle = styled.div`
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 40px;
+  background-color: #203d7a;
+
+  @media screen and (max-width: 768px) {
+    width: 80px;
+    height: 80px;
+  }
+`;
+
+const Icon = styled.img`
+  width: 45%;
+`;
+
+interface Props {
+  icon?: number;
+  title?: string;
+  detail?: string;
+}
+
+const DetailsCard: React.FC<Props> = (props) => {
+  const iconList = [CalendarIcon, ClockIcon, LocationIcon];
+
+  return (
+    <CardContainer>
+      <div>
```

Comment:
Is this outer `div` required?

Reply:
Yeah, so I needed this outer div because the div which serves as the circle shape doesn't retain a perfect circle shape if I don't have an outer div wrapping around it (this was something I had to trial and error with). Maybe there is a better way but it worked at the moment so I just went along with it.